tests/cache_tests.py

```diff
@@ -1,7 +1,9 @@
-from unittest import TestCase
+from unittest import TestCase, skipIf
 from . import utils
 import memcache
+import os
```

Comment:
let's put standard imports at the beginning instead, then third-party modules and then ours. I know we don't do this on all code, but let's do it for new code at least.